layouts/shortcodes/clients.html

```diff
@@ -43,19 +30,24 @@ <h2 id="libraries-{{ urlize . }}">{{ . }}</h2>
     {{ end }}
     {{ range $.Site.Data.clients.list }}
         {{ if eq $library .library }}
-        <li>
-            <a href="{{ if .library_url }}{{ .library_url }}{{ else }}{{ .url }}{{ end }}">{{ .name }}</a>
+        <li {{ if not .acme_v2 }}style="color:grey"{{ end }}>
+            <a {{ if not .acme_v2 }}style="color:grey"{{ end }} href="{{ if .library_url }}{{ .library_url }}{{ else }}{{ .url }}{{ end }}">{{ .name }}</a>
             {{ if .library_comments }}
                 {{ .library_comments | markdownify }}
             {{ else if .comments }}
                 {{ .comments | markdownify }}
             {{ end }}
+            {{ if not .acme_v2 }}<a href="#acme_v1">*</a>{{ end }}
         </li>
         {{ end }}
     {{ end }}
     </ul>
 {{ end }}
 
+<div id="acme_v1">
+    * Clients in <span style="color:grey">grey</span> doesn't support the new ACME v2 API, only the old, deprecated API ACME v1. They will be removed from this list soon.
+</div>
```

Comment:
Reading this again I think it might be worth adding a note here about submitting a PR to `data/clients.json` if a client is misrepresented as ACME v1. That gives folks a nudge to fix it themselves vs just opening an issue/forum thread about it.

Reply:
Do you have a suggestion? I was thinking about:

>Note for clients developers: If your client is compatible with ACME v2 but appears in <span style="color:grey">grey</span>, please submit a pull-request to update `data/clients.json` on https://github.com/letsencrypt/website

Reply:
That sounds great. Thanks @tdelmas